Source/Core/Logging/FramesLogEvent.swift

```diff
@@ -56,6 +58,8 @@ enum FramesLogEvent: Equatable, PropertyProviding {
             return "payment_form_presented"
         case .paymentFormSubmitted:
             return "payment_form_submitted"
+        case .paymentFormOutcome:
+            return "payment_form_outcome"
```

Comment:
Can we rename this to `payment_form_submitted_result` ?

Reply:
✅